From 18dfd8bd7f5d89efc0765451ae24586c9f3d75dc Mon Sep 17 00:00:00 2001
From: Martin Kepplinger <martink@posteo.de>
Date: Thu, 31 Aug 2017 07:06:54 +0200
Subject: [PATCH] remove dependency on pkg-config

let's just assume the SDL2 library is called libSDL2 and can be linked against

Signed-off-by: Martin Kepplinger <martink@posteo.de>
---
 configure.ac      | 4 +---
 tests/Makefile.am | 4 ++--
 2 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index eba0cb4..fce856d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -175,9 +175,7 @@ AC_ARG_WITH(sdl2,
 AM_CONDITIONAL(SDL, test x"$with_sdl2" = "xyes")
 AS_IF([test "x$with_sdl2" = "xyes"], [
   AC_DEFINE(HAVE_SDL2,,[Use SDL2.])
-  PKG_CHECK_MODULES([SDL2], [sdl2], [], [
-    AC_CHECK_LIB([SDL2], [SDL_Init], [], [exit 1])
-  ])
+  AC_CHECK_LIB([SDL2], [SDL_Init], [], [exit 1])
 ])
 
 # libts Library versioning
diff --git a/tests/Makefile.am b/tests/Makefile.am
index 95f5843..f2c3a6d 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -53,7 +53,7 @@ endif
 
 if SDL
 ts_test_mt_SOURCES	= ts_test_mt_sdl.c sdlutils.c sdlutils.h
-ts_test_mt_LDADD	= $(top_builddir)/src/libts.la @SDL2_LIBS@
+ts_test_mt_LDADD	= $(top_builddir)/src/libts.la -lSDL2
 else
 ts_test_mt_SOURCES	= ts_test_mt.c testutils.c testutils.h fbutils.h font_8x8.c font_8x16.c font.h
 ts_test_mt_LDADD	= $(top_builddir)/src/libts.la
@@ -78,7 +78,7 @@ ts_print_raw_LDADD	= $(top_builddir)/src/libts.la
 
 if SDL
 ts_calibrate_SOURCES	= ts_calibrate_sdl.c ts_calibrate.h ts_calibrate_common.c sdlutils.c sdlutils.h
-ts_calibrate_LDADD	= $(top_builddir)/src/libts.la @SDL2_LIBS@
+ts_calibrate_LDADD	= $(top_builddir)/src/libts.la -lSDL2
 else
 ts_calibrate_SOURCES	= ts_calibrate.c ts_calibrate.h ts_calibrate_common.c fbutils.h testutils.c testutils.h font_8x8.c font_8x16.c font.h
 ts_calibrate_LDADD	= $(top_builddir)/src/libts.la
-- 
2.11.0

